### PR TITLE
headless-browser+Ladybird: Move the running of layout tests to headless-browser

### PR DESCRIPTION
**Pull request by trflynn89**
Rather than having a one-off mode in Ladybird, let's use headless-browser for headlessly running these layout tests

### CODE REVIEW

File: Userland/Utilities/headless-browser.cpp [hunk 1]
```diff
@@ -171,6 +186,18 @@ static ErrorOr<NonnullRefPtr<Core::Timer>> load_page_for_screenshot_and_exit(Cor
...
+{
+    if (Core::DeprecatedFile::exists(url))
+        return URL::create_with_file_scheme(Core::DeprecatedFile::real_path_for(url));
+
+    URL formatted_url { url };
+    if (!formatted_url.is_valid())
+        formatted_url = TRY(String::formatted("http://{}", url));
+
+    return formatted_url;
+}
```
Comment (linusg) [inline Userland/Utilities/headless-browser.cpp:199]: That seems unrelated to the commit? ("headless-browser: Add a mode to dump the layout tree after load finish")

Comment (trflynn89) [reply Userland/Utilities/headless-browser.cpp:199]: Sure I can separate it - this was basically "copy-paste layout tree dump from ladybird" and that included this.

**Review by linusg — APPROVED** [review]
Nice! Letting CI run to completion for this one, but changes LGTM.
